androidApp/src/test/kotlin/dev/johnoreilly/confetti/SpeakerDetailsScTest.kt

```diff
@@ -0,0 +1,54 @@
+package dev.johnoreilly.confetti
+
+import dev.johnoreilly.confetti.di.KoinTestApp
+import dev.johnoreilly.confetti.fragment.SpeakerDetails
+import dev.johnoreilly.confetti.speakerdetails.SpeakerDetailsView
+import dev.johnoreilly.confetti.test.screenshot.createScreenshotTestRule
+import dev.johnoreilly.confetti.ui.ConfettiTheme
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.koin.test.KoinTest
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.GraphicsMode
+
+// TODO: Add qualifiers
+@RunWith(RobolectricTestRunner::class)
+@Config(
+    application = KoinTestApp::class,
+    sdk = [30],
+)
+@GraphicsMode(GraphicsMode.Mode.NATIVE)
+class SpeakerDetailsScTest : KoinTest {
```

Comment:
I should probably make some base or something here, but I want to generalise it for like tablet, phones and wear maybe. So I'm not quite sure on that yet